src/entities/swap/swaps/v3/index.ts

```diff
@@ -532,6 +532,7 @@ export class SwapV3 implements SwapBase {
                             return {
                                 pool: pool,
                                 tokenOut: p.tokens[i + 1].address,
+                                isBuffer: false,
```

Comment:
Probably wrong to hardcode these as false, but I'm not sure how to go about writing the logic to toggle `isBuffer` for the swap path steps

Reply:
Yeah, hardcoding is fine for now, we'll have to figure out boosted pools later.